### PR TITLE
fix(ContactsList): Show avatar even if name is long

### PR DESCRIPTION
**Pull request by drazik**
Avatar shrinked to 0px when the name was long. It doesn't shrink anymmore.

### CODE REVIEW

File: react/ContactsList/styles.styl [hunk 1]
```diff
@@ -59,6 +59,9 @@ $reset-list
 .contact-lastname
     font-weight bold
 
+.contact-avatar
+    flex-shrink 0
```
Comment (ptbrowne) [inline react/ContactsList/styles.styl]: It should be directly on the avatar right ? An avatar should never shrink IMHO.

Comment (ptbrowne) [reply react/ContactsList/styles.styl]: cc @y-lohse

Comment (y-lohse) [reply react/ContactsList/styles.styl]: Not sure. It should have a fixed size, but it already has a fixed width and height. I don't think the component itself should define how it behaves when it's put inside a flex layout. There are other ways to change the size of a component (grid layout, absolute positions, etc), imo it's the responsibility of the component user to handle those cases, like it's been done here.

Comment (drazik) [reply react/ContactsList/styles.styl]: Yes I don't think the Avatar should know how it should behave in all contexts in which it can be rendered.

Comment (ptbrowne) [reply react/ContactsList/styles.styl]: OK good point, thanks !

**Pull request merged**